packages/rest/src/parser.ts

```diff
@@ -78,29 +85,47 @@ export async function parseOperationArgs(
 async function loadRequestBodyIfNeeded(
   operationSpec: OperationObject,
   request: Request,
-): Promise<MaybeBody> {
-  if (!operationSpec.requestBody) return Promise.resolve();
+): Promise<RequestBody> {
+  if (!operationSpec.requestBody) return Promise.resolve({value: undefined});
 
   const contentType = getContentType(request);
   debug('Loading request body with content type %j', contentType);
+
+  if (
+    contentType &&
+    contentType.startsWith('application/x-www-form-urlencoded')
+  ) {
+    const body = await parseFormBody(request).catch((err: HttpError) => {
+      debug('Cannot parse request body %j', err);
+      err.statusCode = 400;
```

Comment:
This branch is still not covered by any test, see https://coveralls.io/builds/19498795/source?filename=packages%2Frest%2Fsrc%2Fparser.ts

Please add a new test. AFAICT, `body/form` returns error only when the request body is larger than the configured limit (1MB by default) or when an unsupported content-encoding is used. I guess our best option is to explicitly set `Content-Length` header of the outgoing request to a value larger than 1MB.

Relevant source code:
- https://github.com/Raynos/body/blob/master/index.js
- https://github.com/stream-utils/raw-body/blob/master/index.js